### PR TITLE
WIP: update cloud setup guide

### LINKED ISSUE #230: Rework setup guides
We currently have a handful of setup guides for dask/xarray/pangeo for Cheyenne and Geyser/Caldera. These are quite out of date now and seem to be causing potential users/adopters all sorts of problems.  I am proposing that we create more general setup guides:

### HPC systems
In this, we'll discuss how to:
 1. setup a conda environment,
 1. launch jupyter,
 1. connect via an ssh tunnel,
 1. launch dask distributed via dask-jobqueue or dask-mpi

I'm open to suggestions on how much detail we really want to include here. These steps can be somewhat machine specific so we'll want to separate those parts. 

### Cloud systems
In this, we'd want to leverage the zero-to-jupyterhub but cover the pangeo specific bits. There may also be an argument for contributing some "cloud for dummies" content to zero-to-jupyterhub since we've been developing some new user know-how in this area.

### PR DESCRIPTION
**Pull request by rabernat**
This is the start of an update to the cloud setup guide. I worked on it a bit today with @kaipak...based on the tutorial given to me by @tjcrone. More commits coming soon.

Closes #230.

### CODE REVIEW

File: docs/setup_guides/cloud.rst [hunk 1]
```diff
@@ -1,85 +1,100 @@
...
-    $ git checkout pangeo
-    $ python setup.py install
+  #!/bin/bash
 
+  set -e
 
-Create and launch your named cluster
-------------------------------------
+  EMAIL='ryan.abernathey@gmail.com'
+  PROJECTID='pangeo-181919'
```
Comment (jhamman) [inline docs/setup_guides/cloud.rst]: probably best to anonymize these two values

**Comment by rabernat** [comment]
This is messy but better than nothing.

Would appreciate a review by @tjcrone, @jacobtomlinson, @mrocklin, or anyone else who has grappled with cloud deployment.

**Comment by tjcrone** [comment]
There is a step 1.5, which would [include](https://github.com/tjcrone/pangeo/blob/gcp/gce/deploy_gcp#L7):

0) create new project if required (recommended), and switch to it
1) in APIs and Services enable the Kubernetes Engine API for the project
2) run some gcloud setup/auth:
```
gcloud config set container/new_scopes_behavior true
gcloud auth login
```

**Comment by rabernat** [comment]
Thanks Tim.

**Pull request merged**